src/asqi/workflow.py

```diff
@@ -865,6 +869,31 @@ def save_results_to_file_step(results: Dict[str, Any], output_path: str) -> None
         DBOS.logger.error(f"Invalid results data cannot be saved to {output_path}: {e}")
 
 
+@DBOS.step()
+def save_container_results_to_file_step(
+    container_results: List[Dict[str, Any]], output_path: str
+) -> None:
+    """Save container results to a JSON file."""
+    logs_path = os.getenv("LOGS_PATH", "logs")
+    try:
+        Path(logs_path).mkdir(exist_ok=True)
```

Comment:
One more comment - perhaps we should add `logs_path` as an input param instead to be consistent with output_path.